distributed/cluster.py

```diff
@@ -90,7 +91,7 @@ def async_ssh(cmd_dict):
 
     print('[ {label} ] : {cmd}'.format(label = cmd_dict['label'],
                                        cmd = cmd_dict['cmd']))
-    stdin, stdout, stderr = ssh.exec_command('$SHELL -i -l -c \'' + cmd_dict['cmd'] + '\'', get_pty = True)
+    stdin, stdout, stderr = ssh.exec_command('$SHELL -i -c \'' + cmd_dict['cmd'] + '\'', get_pty = True)
```

Comment:
A similar implementation can be found in ipyparallel
(https://github.com/ipython/ipyparallel/blob/1509839b69b236cca08407ab85779d04b31e8217/ipyparallel/apps/launcher.py#L64-L68)